docs/source/deployorderer/ordererdeploy.md

```diff
@@ -0,0 +1,272 @@
+
+```
+cd bin
+./orderer start
+```
+
+When the orderer starts successfully, you should see a message similar to:
+
+```
+INFO 019 Starting orderer:
+INFO 01a Beginning to serve requests
+```
+
+You have successfully started one node, you now need to repeat these steps to configure and start the other two orderers. When a majority of orderers are started, a Raft leader is elected. You should see something similar to the following output:
+```
+INFO 01b Applied config change to add node 1, current nodes in channel: [1] channel=syschannel node=1
+INFO 01c Applied config change to add node 2, current nodes in channel: [1 2] channel=syschannel node=1
+INFO 01d Applied config change to add node 3, current nodes in channel: [1 2 3] channel=syschannel node=1
+INFO 01e raft.node: 1 elected leader 2 at term 11 channel=syschannel node=1
+INFO 01f Raft leader changed: 0 -> 2 channel=syschannel node=1
+```
+
+## Next steps
+
+Your ordering service is started and ready to order blocks. Working together to form the ordering service, the orderers receive transactions from application clients and package the transactions into a well-defined sequence of blocks. These blocks form the blockchain and are ultimately distributed to all peers on the channel to be committed to their ledger.
+
+Ultimately though, you may need to add or remove orderers from the consenter set, or other organizations may want to contribute their own orderers to the cluster. See the topic on ordering service [reconfiguration](../raft_configuration.html#reconfiguration) for considerations and instructions.
+
+Finally, your ordering service includes a consortium of members as defined in the `Organization` section of the channel configuration. Additional application channels can be created for custom combinations of ordering service consortium members to transact privately by using the `cfgtxgen` command and the `configtx.yaml` file to generate the genesis block for the new channel.
+
+## Troubleshooting
+
+### When you start the orderer, it fails with the following error:
+```
+ERRO 001 failed to parse config:  Error reading configuration: Unsupported Config Type ""
+```
+
+**Solution:**
+Your `FABRIC_CFG_PATH` is not set. Run the following command to set it to the location of your `orderer.yaml` file.
+
+```
+export FABRIC_CFG_PATH=<PATH_TO_ORDERER_YAML>
+```
+
+### When you start the orderer, it fails with the following error:
+```
+PANI 003 Failed to setup local msp with config: administrators must be declared when no admin ou classification is set
+```
+
+**Solution:**   
+Your local MSP definition is missing the `config.yaml` file. Create the file and copy it into the local MSP `/msp` folder of orderer. See the [Fabric CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html#nodeous) documentation for more instructions.
```

Comment:
I think I noted this in the channel participation tutorial as well but which `config.yaml` are we referring to here?

Reply:
`config.yaml` is a file that enables Node OUs when it's included in an MSP.